src/esp/sim/Simulator.cpp

```diff
@@ -888,6 +887,11 @@ bool Simulator::recomputeNavMesh(nav::PathFinder& pathfinder,
       }
     }
   }
+  ESP_CHECK(joinedMesh->vbo.size() > 0,
+            "::recomputeNavMesh: "
+            "Unable to compute a navmesh upon a non-existent mesh - "
+            "the underlying joined collision mesh has no vertices. This is "
+            "probably due to the current scene being NONE. Aborting");
```

Comment:
...or is it better to have ESP_CHECK for this, which will throw in python and can be trapped in a try if necessary.

Reply:
I like this approach, thanks for adding this check. I've seen the resulting issue a couple times and the current crash message response is non-obvious.